covid_api/api/utils.py

```diff
@@ -679,3 +687,48 @@ def get_custom_cmap(cname) -> Dict:
 
 
 ColorMapName = Enum("ColorMapNames", [(a, a) for a in COLOR_MAP_NAMES])  # type: ignore
+
+
+def planet_mosaic_tile(scenes, x, y, z):
+    """return a mosaicked tile for a set of planet scenes"""
+    mosaic_tile = np.zeros((4, 256, 256), dtype=np.uint8)
+    for scene in scenes:
+        api_num = math.floor(random.random() * 3) + 1
+        url = f"https://tiles{api_num}.planet.com/data/v1/PSScene3Band/{scene}/{z}/{x}/{y}.png?api_key={PLANET_API_KEY}"
+        r = requests.get(url)
+        with MemoryFile(BytesIO(r.content)) as memfile:
+            with memfile.open() as src:
+                data = src.read()
+                # any place we don't have data yet, add some
+                mosaic_tile = np.where(
+                    mosaic_tile[3] == 0, mosaic_tile + data, mosaic_tile
+                )
+
+        # if the tile is full, stop
+        if np.count_nonzero(mosaic_tile[3]) == mosaic_tile[3].size:
```

Comment:
can't you use `if mosaic_tile[3].all()`

Reply:
I think it's marginally slower but yes seems clearer